examples/GainPlugin/Source/jsui/src/index.js

```diff
@@ -1,4 +1,4 @@
-import "juce-blueprint/cjs/lib/polyfill";
+import "juce-blueprint/src/lib/polyfill";
```

Comment:
Should be changed to `juce-blueprint/dist/lib/polyfill` unless there is a better way to do this? I just need to sort out copy step in webpack.